test/Index/Store/unit-one-file-multi-file-invocation.swift

```diff
@@ -7,16 +7,10 @@
 
 // CHECK: 00-output_for_index
 // CHECK: DEPEND START
-// CHECK: Unit | system | Swift | [[MODULE:.*[/\\]Swift[.]swiftmodule([/\\].+[.]swiftmodule)?]] | [[SWIFT:.+[.]swiftmodule-[A-Z0-9]*]]
+// CHECK: Unit | system | Swift | [[MODULE:.*[/\\]Swift[.]swiftmodule([/\\].+[.]swiftmodule)?]]
```

Comment:
Shouldn't we still be indexing `Swift` here?

Reply:
With a development compiler the stdlib is next to the compiler and not in the SDK, so not anymore. When using a compiler from Xcode and an stdlib from the SDK it should still index the stdlib as expected.

Reply:
Is there any way to also include the next-to-compiler libs?

Reply:
It looks like that's a runtime library import path. It took me a while to track that down. It makes sense, I'll look into adding them if you think it's appropriate.

Reply:
It's set by lit.cfg via an env var. It may be relevant only to the test suite.
```
lit.py: swift/test/lit.cfg:2130: note: Testing with the just-built libraries
lit.py: swift/test/lit.cfg:2132: note: Library load path: build/Ninja-RelWithDebInfoAssert/swift-macosx-arm64/lib/swift/macosx
```

Reply:
As discussed, I added an exception to always index the stdlib.